DirectEmployers/MyJobs
# [PD-2846] add remove all button to tags widget

## What's this PR do?
This PR adds a remove all button to the tags widget inside of NUO

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of NUO and go to create a partner or edit an existing partner or contact. You can start with the section that says tags and click on it to bring up the tags widget

## How should this be manually tested?
Pull down the repository and go to the NUO application. Partners and Contacts will have a tag section when filling in information to pull up the tags widget. If you hit the select all button for the tags and pull the box back up you will see that there is a remove all button for removing all of the tags. If you click the button it will put the tags back inside of the box and remove them all.

## Any background context you want to provide?
Might need assistance from Jonathan or anyone else if there are suggestions for making the remove all button functionality different
